src/target/apple.rs

```diff
@@ -0,0 +1,19 @@
+use super::TargetInfo;
+
+impl TargetInfo {
+    pub(crate) fn apple_sdk_name(&self) -> &'static str {
+        match (&*self.os, &*self.abi) {
+            ("macos", "") => "macosx",
+            ("ios", "") => "iphoneos",
```

Comment:
Fun fact: x86_64-apple-ios is actually iphonesimulator.

Reply:
oops 😅 looks like we have to update this

Reply:
https://github.com/rust-lang/cc-rs/blob/7a786c5e3dc149f32e565360dd8c3e43748592f5/src/target/apple.rs#L8

Is checking for `abi = "sim"` enough?

Reply:
Yes, it is, `rustc` [sets the correct ABI](https://github.com/rust-lang/cc-rs/blob/7a786c5e3dc149f32e565360dd8c3e43748592f5/src/target/generated.rs#L2811).